### ISSUE TITLE
Components wrapped inside a fragment do not have Dividers

### ISSUE THREAD

**Issue by petewins** [formaat-design/reshaped#226]
**Describe the bug**
Components wrapped inside a fragment do not have Dividers inside a View. Not sure if this is the intended behavior given how the Fragment wrapper is initially counted as one component but I noticed this behavior when I had a Fragment with dynamic number of children

**To Reproduce**
https://codesandbox.io/p/devbox/reshaped-view-divider-demo-8sprjr?file=%2Fapp%2Fpage.tsx%3A12%2C1

**Expected behavior**
Expect Children of the View to have dividers, including Child components with fragments

**Screenshots**
<img width="385" alt="image" src="https://github.com/formaat-design/reshaped/assets/19542805/96eb9b31-eca7-49a4-8929-6ab43ecee10b">
<img width="613" alt="image" src="https://github.com/formaat-design/reshaped/assets/19542805/665a0c48-4e93-4a44-a1ea-1c70e49cb1aa">

**Environment (please complete the following information):**
 - OS: OSX 14.3.1
 - Browser: Chrome

**Additional context**

**Comment by blvdmitry** [comment]
Should be fixed in 2.10.5

```
<View direction="row" gap={4} divided>
	<Placeholder>Item 1</Placeholder>
	<>
		<Placeholder>Item 2</Placeholder>
		<Placeholder>Item 3</Placeholder>
	</>
</View>
```

![Image](https://github.com/formaat-design/reshaped/assets/887379/91fb6d79-b96b-443b-ab7d-734dfacefbea)

**Comment by petewins** [comment]
Thank you 💛
